### PR TITLE
Public Schedule: Only show days with sessions scheduled

### LINKED ISSUE #6092: Public Schedule: Only show days with sessions scheduled
For events that take place over several days please only show days that have sessions scheduled. Please take care that the scheduler used to put the sessions into the day used by the organizer is not changed.

e.g. compare https://eventyay.com/e/ae73f280/schedule

Screenshot from 2020-12-20 00-58-38

### PR DESCRIPTION
**Pull request by Draco9421**
Screenshot from 2020-12-20 20-39-22

Fixes #6092

#### Changes proposed in this pull request:

-
-
-

#### Checklist

- [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/controllers/public/sessions.js [hunk 1]
```diff
@@ -16,10 +16,9 @@ export default class SessionsController extends Controller {
   @computed('model.event.startsAt', 'model.event.endsAt', 'timezone')
   get allDates() {
     const arr = [];
-    const difference = (this.model.event.endsAt).diff(this.model.event.startsAt, 'days');
-    for (let i = 0; i <= Math.abs(difference); i++) {
-      arr.push(moment.tz(this.model.event.startsAt, this.timezone).add(i, 'days').toISOString());
-    }
+    this.model.event.sessions.toArray().forEach(Dates => {
+      arr.push(moment.tz(Dates.startsAt, this.timezone).toISOString());
+    });
```
Comment (maze-runnar) [inline app/controllers/public/sessions.js:21]: wrong!

**Comment by Afzal-Ind** [comment]
Hi, I found something like this when I run the deployment. Plz have a look at this.
c2

**Comment by Draco9421** [comment]
> Hi, I found something like this when I run the deployment. Plz have a look at this and try to fix it.
> c2

I think all the sessions of that event are on the same day that's why its looking like that.

File: app/controllers/public/sessions.js (hunk 1 above)
Comment (Draco9421) [reply app/controllers/public/sessions.js:21]: Please can you tell what's wrong in it sir ? Did I understood the issue wrong.

**Comment by Afzal-Ind** [comment]
> I think all the sessions of that event are on the same day that's why its looking like that.

Then why 13th Dec session is missing. Plz recheck and try again. u can get it.

**Pull request closed without merging**